packages/ui-components/src/components/PageHeader/PageHeader.stories.tsx

```diff
@@ -64,24 +69,34 @@ export const WithHeading: Story = {
   },
 }
 
+export const WithHeadingAndOptions: Story = {
```

Comment:
Options is the internal name. I think it'd be better to say "children" here because that's how users have to use it
```suggestion
export const WithHeadingAndChildren: Story = {
```

Reply:
Agreed.